ioos_tools/ioos.py

```diff
@@ -541,7 +541,7 @@ def load_ncs(config):
         if 'OBS_DATA' in fname:
             continue
         else:
-            model = fname.split('.')[0].split('/')[-1].split('-')[-1]
+            model= os.path.splitext(os.path.split(fname)[-1])[0].split('-')[-1]
```

Comment:
@rsignell-usgs just to let you know this is the issue we had in the notebook that resulted in the `KeyError` later.
Instead of spiting at the `.` and `/` I should be using `os.path` `splitext` and `split for multi-platform support.